xwiki-commons-core/xwiki-commons-observation/xwiki-commons-observation-api/src/main/java/org/xwiki/observation/event/ApplicationEvent.java

```diff
@@ -20,7 +20,7 @@
 package org.xwiki.observation.event;
 
 /**
- * All Event types must implement this interface.
+ * All ApplicationEvent types must implement this interface.
```

Comment:
Why?

Reply:
also what is an `ApplicationEvent`?

Reply:
My point is that I think you're doing a PR without understanding the code and the underlying things being done.

Reply:
Since they should let know that they are child of this interface which is a parent.
I might be wrong, can you correct me?